### PR TITLE
Setup a matrix bot

### PR DESCRIPTION
**Pull request by heurtematte**
The steps follow by the script:
* Test connexion via tunnel
* test login via matrix-token in cbi config file
* Create Credential for Matrix bot user (in pass)
* Create Matrix bot user (in matrix)
* Validate Matrix bot user consent
* Get Matrix bot user token (from matrix and register in pass)
* Join bot to matrix room (specific room request by project)
* Leave #eclipse and #eclipsefdn-project default spaces (added by default by matrix configuration)
* Add Matrix bot token to Jenkins credentials

### CODE REVIEW

File: pass/add_creds.sh [hunk 1]
```diff
@@ -167,6 +167,16 @@ github_ssh() {
   ssh_keys "${project_name}" "${site}" "${user}"
 }
 
+matrix() {
```
Comment (fredg02) [inline pass/add_creds.sh:170]: "matrix" command should be added to the help() function.

Comment (heurtematte) [reply pass/add_creds.sh:170]: done!

Comment (fredg02) [reply pass/add_creds.sh:170]: Are you sure you have pushed the change? I only see: https://github.com/eclipse-cbi/ci-admin/compare/b685abc222264756d0b1f4215ada298da4017c61..87ec1a878d8e4ebb785fc4dda324bd0defbfbd37

Comment (fredg02) [reply pass/add_creds.sh:171]: Looks better now! :+1:

File: pass/add_creds.sh [hunk 2]
```diff
@@ -107,6 +107,7 @@ help() {
   printf "gerrit\t\t\tCreate Gerrit credentials (SSH keypair).\n"
   printf "github\t\t\tCreate GitHub credentials (username/password).\n"
   printf "github_ssh\t\tCreate SSH credentials for GitHub (SSH keypair).\n"
+  printf "matrix\t\tCreate Matrix credentials for (username/password).\n"
```
Comment (fredg02) [inline pass/add_creds.sh:110]: Sorry to nitpick. Text should probably be "Create Matrix credentials for chat.eclipse.org (username/password)"

**Comment by fredg02** [comment]
@heurtematte I'd recommend avoiding force-pushes to fix issues in a PR. With force-pushes it's harder to track incremental changes.

**Comment by fredg02** [comment]
Nope

**Pull request merged**